GameData/StationPartsExpansionRedux/Patches/SSPXR-B9TankTypes-Shelter.cfg

```diff
@@ -0,0 +1,93 @@
+B9_TANK_TYPE
+{
+	name = SSPX_Shelter_Monopropellant
+	tankMass = 0.00053333333 //based on stock FL-R1 2.5m RCS tank
+	tankCost = 0
+	RESOURCE
+	{
+		name = MonoPropellant
+		unitsPerVolume = 1
+	}
+}
+
+B9_TANK_TYPE
+{
+	name = SSPX_Shelter_LFO
+	tankMass = 0.000625
+	tankCost = 0
+	RESOURCE
+  {
+    name = LiquidFuel
+    unitsPerVolume = 0.45
+  }
+  RESOURCE
+  {
+    name = Oxidizer
+    unitsPerVolume = 0.55
+  }
+}
+
+B9_TANK_TYPE
+{
+	name = SSPX_Shelter_LiquidFuel
+	tankMass = 0.000625
+	tankCost = 0
+	RESOURCE
+  {
+    name = LiquidFuel
+    unitsPerVolume = 1
+  }
+}
+
+B9_TANK_TYPE
+{
+	name = SSPX_Shelter_Oxidizer
+	tankMass = 0.000625
+	tankCost = 0
+	RESOURCE
+  {
+    name = Oxidizer
+    unitsPerVolume = 1
+  }
+}
+
+B9_TANK_TYPE:NEEDS[CryoTanks]
+{
+	name = SSPX_Shelter_LH2
+	tankMass = 0.00010627500
+	tankCost = 0
+	RESOURCE
+  {
+    name = LqdHydrogen
+    unitsPerVolume = 7.5
+  }
+}
+
+B9_TANK_TYPE:NEEDS[CryoTanks]

```

Comment:
If you are using `NEEDS[CryoTanks]` I would recommend just using that definition rather than creating a new one

Reply:
Good point; change made.